Ginger/GingerCoreNET/Run/Context.cs

```diff
@@ -39,11 +42,12 @@ public GingerRunner Runner
                 if(mRunner != value)
                 {
                     mRunner = value;
+                    SetAgent();
```

Comment:
remove SetAgent we should not have dependency between properties

Reply:
Removed